### PR TITLE
Fixes a bunch of spelling mistakes

### PR DESCRIPTION
**Pull request by Contrabang**
# About the pull request

Most of the changes here are minor, mostly to comments or descriptions. Here is the full list.

- recieve -> receive
- percieve -> perceive
- alot -> a lot
- atleast -> at least
- succesfully -> successfully
- seperate -> separate
- noticable -> noticeable
- concious -> conscious
- enviroment -> environment
- existance -> existence
- foward -> forward
- interupted -> interrupted
- independant -> independent
- guarenteed -> guaranteed
- overriden -> overridden
- occuring -> occurring
- occured -> occurred
- transciever -> transceiver
- maintenace -> maintenance

# Explain why it's good for the game
Makes it easier to read things

# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

Code compiles and runs

</details>

# Changelog
:cl:
spellcheck: Fixed a few uncommon typos across the entire codebase
/:cl:

### CODE REVIEW

File: code/datums/redis/callbacks/_redis_callback.dm [hunk 1]
```diff
@@ -19,7 +19,7 @@
  * * message - The message received on the redis channel.
  */
 /datum/redis_callback/proc/on_message(message)
-	CRASH("on_message not overriden for [type]!")
+	CRASH("on_message not overridden for [type]!")
```
Comment (Contrabang) [inline code/datums/redis/callbacks/_redis_callback.dm:22]: This change may be notable if specific errors are being tracked across multiple rounds in some kind of database, if so, I can change this back.

**Comment by LC4492** [comment]
Also, the maintenance semiotic is also miss-spelled, being called 'maintenace' instead of the correct name. You could look at that too in the meanwhile.

File: code/datums/redis/callbacks/_redis_callback.dm (hunk 1 above)
Comment (Birdtalon) [reply code/datums/redis/callbacks/_redis_callback.dm:22]: This is fine.

**Review by Birdtalon — APPROVED** [review]
Sure.

**Pull request merged**